### PR TITLE
My addition

### PR DESCRIPTION
**Pull request by shadeslayer93**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2026)

### CODE REVIEW

File: Malayalam/malayalam.md [hunk 1]
```diff
@@ -1 +1,3 @@
-ഞാൻ ഒരു മലയാളി. ഒരു സാഹസിക യാത്രയുടെ തുടക്കം.
\ No newline at end of file
+ഞാൻ ഒരു മലയാളി. ഒരു സാഹസിക യാത്രയുടെ തുടക്കം.
+
+ഇത് ഒരു നല്ല [കുട്ടിയുടെ](Malayalam/boy/life.md) കഥയാണ്
```
Comment (casey-collab) [inline Malayalam/malayalam.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Malayalam/boy/life.md [hunk 2]
```diff
@@ -0,0 +1 @@
+ഈ കുട്ടിയുടെ പേര്?
```
Comment (casey-collab) [inline Malayalam/boy/life.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by shadeslayer93** [comment]
Please, could you check, whether this is working @casey-collab

**Comment by casey-collab** [comment]
Looks great! :dart:

**Pull request merged**